pkg/services/store/entity/sqlstash/sql_storage_server.go

```diff
@@ -1106,12 +1179,356 @@ func (s *sqlEntityServer) List(ctx context.Context, r *entity.EntityListRequest)
 	return rsp, err
 }
 
-func (s *sqlEntityServer) Watch(*entity.EntityWatchRequest, entity.EntityStore_WatchServer) error {
+func (s *sqlEntityServer) Watch(r *entity.EntityWatchRequest, w entity.EntityStore_WatchServer) error {
```

Comment:
i'm seeing this error with `kubernetesAggregator = true`:

```
ERROR[03-04|15:34:04] pkg/mod/k8s.io/client-go@v0.29.2/tools/cache/reflector.go:229: Failed to watch *v0alpha1.ExternalName: failed to list *v0alpha1.ExternalName: Internal error occurred: invalid key (expecting at least 3 parts): /service.grafana.app/externalnames logger=grafana-apiserver
```

Reply:
That's the issue we know about with non-namespaced keys